### PR TITLE
sql-cache tool and update sample version

### PR DESCRIPTION
**Pull request by iscifoni**
Update document, added sql-cache tool.

Update sample to version 1.0.0

Document sql-cache tool #1412

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@kichalla  please review.

File: aspnet/performance/caching/distributed/sample/src/DistCacheSample/project.json [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
   },
 
-  "commands": {
-    "web": "Microsoft.AspNet.Server.Kestrel"
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "1.0.0-preview2-final",
+      "imports": "portable-net45+win8+dnxcore50"
+    },
+    "Microsoft.Extensions.Caching.SqlConfig.Tools": "1.0.0-*"
```
Comment (kichalla) [inline aspnet/performance/caching/distributed/sample/src/DistCacheSample/project.json]: Change the version to `1.0.0-preview2-final`

File: aspnet/performance/caching/distributed/sample/src/DistCacheSample/Startup.cs [hunk 2]
```diff
@@ -45,54 +44,32 @@ public void ConfigureStagingServices(IServiceCollection services)
...
-            new SqlServerCache(new CacheOptions(new SqlServerCacheOptions()
+          
+            services.AddDistributedSqlServerCache(conf =>
             {
-                ConnectionString = @"Data Source=(localdb)\v11.0;Initial Catalog=DistCache;Integrated Security=True;",
-                SchemaName = "dbo",
-                TableName = "TestCache"
-            })));
+                conf.ConnectionString = @"Data Source=(localdb)\v11.0;Initial Catalog=DistCache;Integrated Security=True;";
+                    conf.SchemaName = "dbo";
```
Comment (kichalla) [inline aspnet/performance/caching/distributed/sample/src/DistCacheSample/Startup.cs]: Fix identation

File: aspnet/performance/caching/distributed/sample/src/DistCacheSample/Startup.cs [hunk 3]
```diff
@@ -45,54 +44,32 @@ public void ConfigureStagingServices(IServiceCollection services)
         /// <param name="services"></param>
         public void ConfigureProductionServices(IServiceCollection services)
         {
-            // Use SQL Server
-            services.AddSingleton<IDistributedCache>(serviceProvider =>
-            new SqlServerCache(new CacheOptions(new SqlServerCacheOptions()
+          
+            services.AddDistributedSqlServerCache(conf =>
```
Comment (kichalla) [inline aspnet/performance/caching/distributed/sample/src/DistCacheSample/Startup.cs]: nit: rename `conf` to `options`

File: aspnet/performance/caching/distributed/sample/src/DistCacheSample/Startup.cs [hunk 4]
```diff
@@ -30,13 +30,12 @@ public void ConfigureDevelopmentServices(IServiceCollection services)
...
         {
-            // use Redis
-            services.AddSingleton<IDistributedCache>(serviceProvider => 
-                new RedisCache(new RedisCacheOptions
-                {
-                    Configuration = "localhost",
-                    InstanceName = "SampleInstance"
-                }));
+
+            services.AddDistributedRedisCache(opt =>
```
Comment (kichalla) [inline aspnet/performance/caching/distributed/sample/src/DistCacheSample/Startup.cs]: nit: rename `opt` to `options`

**Comment by kichalla** [comment]
:shipit: after addressing minor comments

**Pull request merged**